docs/administration/configuration/system-properties.md

```diff
@@ -18,12 +18,10 @@ java -server -Dserver.servlet.session.timeout=3600 -Dserver.port=8080 -jar runde
 - `server.port` The HTTP port to use for the server, default "4440"
 - `server.https.port` The HTTPS port to use or the server, default "4443"
 - `server.address` Address/hostname to listen on, default is "localhost"
-- `server.contextPath` Web context path to use, such as "/rundeck". Default is "/". (Pre-Rundeck 3.3.x)
```

Comment:
I think any difference between 3.3.x and pre 3.3.x should be highlighted in the upgrade notes, not buried here @fdevans

Reply:
agreed